server/bundles/io.cloudbeaver.server/schema/service.events.graphqls

```diff
@@ -16,9 +16,13 @@ enum CBServerEventId {
     cb_datasource_folder_updated,
     cb_datasource_folder_deleted,
 
-    cb_rm_resource_created
-    cb_rm_resource_updated
-    cb_rm_resource_deleted
+    cb_rm_resource_created,
+    cb_rm_resource_updated,
+    cb_rm_resource_deleted,
+
+    cb_rm_project_created,
```

Comment:
may be it better  this event as  `cb_rm_project_added`, 
in the case of changing access rights, we add this project to the user, and do not create a new